tasks/main.yml

```diff
@@ -36,29 +36,14 @@
 - name: install docker-rotate cron script
   template: src=docker-rotate.j2 dest=/etc/cron.daily/docker-rotate mode=755
 
-- name: push logrotate config file for docker container json logs
-  template: src=logrotate.conf.j2 dest=/etc/logrotate.d/docker
-  when: installed_docker_version | version_compare('1.8', '<')
-
+# docker log rotation is now managed by Docker (since Docker 1.8). We no longer deal with
```

Comment:
Have we migrated to docker 1.8+ everywhere? Wouldn't want log rotation removed for people still using docker <1.8